app/searchhandler.py

```diff
@@ -38,7 +38,7 @@ async def get(self, request):
 
     await saml.ensure_logged_in(request)
 
-    await clear_stored_search_criteria(session)
+    await clear_stored_search_criteria(session, request)
```

Comment:
If we are passing `request`, we do not need to pass `session`

Reply:
ADjusted to get session from r equest